pkg/plugins/policies/meshretry/api/v1alpha1/validator_test.go

```diff
@@ -342,7 +342,7 @@ violations:
   - field: spec.to[0].default.conf.grpc.retryOn[2]
     message: unknown item 'wrong'`,
 			}),
-			Entry("http retriableHeaders wrong values", testCase{
+			Entry("http retriableRequestHeaders wrong values", testCase{
```

Comment:
As per slack conversation I'd remove the whole entry because we don't want to test this here.